### ISSUE TITLE
makemigration issue

### ISSUE THREAD

**Issue by gt-liu** [divio/django-cms#4138]
On the [Installing django CMS](https://django-cms.readthedocs.org/en/develop/introduction/install.html) doc page it advises us to install Django v1.6.
And on the [Plugins](https://django-cms.readthedocs.org/en/develop/introduction/plugins.html) doc page it says to execute
`python manage.py makemigrations polls`
However, `makemigrations` is not supported in Django v1.6. And when I try to run `python manage.py syncdb` (with the venv activated) it produces the following errors:

```
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/lgt/CMSDemo/lib/python3.4/site-packages/django/core/management/__init__.py", line 399, in execute_from_command_line
    utility.execute()
  File "/Users/lgt/CMSDemo/lib/python3.4/site-packages/django/core/management/__init__.py", line 392, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/lgt/CMSDemo/lib/python3.4/site-packages/django/core/management/base.py", line 242, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/Users/lgt/CMSDemo/lib/python3.4/site-packages/django/core/management/base.py", line 284, in execute
    self.validate()
  File "/Users/lgt/CMSDemo/lib/python3.4/site-packages/django/core/management/base.py", line 310, in validate
    num_errors = get_validation_errors(s, app)
  File "/Users/lgt/CMSDemo/lib/python3.4/site-packages/django/core/management/validation.py", line 34, in get_validation_errors
    for (app_name, error) in get_app_errors().items():
  File "/Users/lgt/CMSDemo/lib/python3.4/site-packages/django/db/models/loading.py", line 196, in get_app_errors
    self._populate()
  File "/Users/lgt/CMSDemo/lib/python3.4/site-packages/django/db/models/loading.py", line 75, in _populate
    self.load_app(app_name, True)
  File "/Users/lgt/CMSDemo/lib/python3.4/site-packages/django/db/models/loading.py", line 99, in load_app
    models = import_module('%s.models' % app_name)
  File "/Users/lgt/CMSDemo/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/Users/lgt/CMSDemo/lib/python3.4/site-packages/djangocms_text_ckeditor/models.py", line 15, in <module>
    from cms.models import CMSPlugin
  File "/Users/lgt/CMSDemo/lib/python3.4/site-packages/cms/models/__init__.py", line 3, in <module>
    from .pagemodel import *  # nopyflakes
  File "/Users/lgt/CMSDemo/lib/python3.4/site-packages/cms/models/pagemodel.py", line 38, in <module>
    class Page(six.with_metaclass(PageMetaClass, MP_Node)):
  File "/Users/lgt/CMSDemo/lib/python3.4/site-packages/django/utils/six.py", line 566, in with_metaclass
    return meta("NewBase", bases, {})
  File "/Users/lgt/CMSDemo/lib/python3.4/site-packages/cms/models/metaclasses.py", line 17, in __new__
    return super_new(cls, name, bases, attrs)
  File "/Users/lgt/CMSDemo/lib/python3.4/site-packages/django/db/models/base.py", line 78, in __new__
    module = attrs.pop('__module__')
KeyError: '__module__'
```

**Comment by yakky** [comment]
There is an inconsistency in the tutorial fixed by #4136
If you installed 1.6, you have to use ``manage.py schemamigration``

**Comment by yakky** [comment]
Closed by #4136

**Comment by gt-liu** [comment]
Oh, thanks!
